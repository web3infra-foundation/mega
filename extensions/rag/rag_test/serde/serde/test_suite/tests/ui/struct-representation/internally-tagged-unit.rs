use serde_derive::Serialize;

#[derive(Serialize)]
#[serde(tag = "type")]
struct U;

fn main() {}
