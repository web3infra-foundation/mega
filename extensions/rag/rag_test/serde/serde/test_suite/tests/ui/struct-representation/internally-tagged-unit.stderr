error: #[serde(tag = "...")] can only be used on enums and structs with named fields
 --> tests/ui/struct-representation/internally-tagged-unit.rs:4:9
  |
4 | #[serde(tag = "type")]
  |         ^^^^^^^^^^^^
